test/prism_regression/def.rb

```diff
@@ -2,16 +2,4 @@
 
 def foo; end
 
-def bar
-  "string1"
-  "string2"
```

Comment:
Method definitions with bodies are already tested in `def_with_body.rb`

Removing those from here makes it possible to test these before statements nodes can be translated.